rental-angular/src/app/components/branches/company-branch.component.ts

```diff
@@ -48,4 +54,33 @@ saveBranch(): void{
 }
 
 
+
+  // getCar(id: number):void {
+  //   this.restService.getBranchCars(id)
+  //     .subscribe(cars => {
+  //       this.cars = cars;
+  //     });
+  // }
+  getCarsId():void{
+    this.restService.getBranchCars(1)  //todo przycisk do wczytywania danych
```

Comment:
masz na sztywno id 1 dlatego pobierają Ci się tylko z jednago oddziału

Reply:
tak, teraz zmieniłem na sztywno, ale gdy:
 zmieniłem na np. branch.id to zaczęło się sypać

Reply:
a branch nie był przypadkiem undefined?

Reply:
tak, był, ale jak się zastosuje przycisk to wtedy pobiera dane z danego oddziału, niestety przez pętle przycisk działa na wszystkie oddziały i wyświetla te same dane jednego oddziały we wszystkich oddziałach

Reply:
Musisz tak wykombinowac aby pobierał tego który został kliknięty [;